src/params.h

```diff
@@ -18,6 +18,7 @@
 #pragma once
 
 #include "wallet.h"
+#include "stratum_server.h"
```

Comment:
No need to include the whole file for just one constant. It makes more sense to move the constant here instead, since it's just a default value for a parameter now.